spec/rubocop/cop/rspec/factory_bot/factory_class_name_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::FactoryBot::FactoryClassName do
+  subject(:cop) { described_class.new }
+
+  context 'when passing block' do
+    context 'when a class' do
+      it 'registers an offense' do
```

Comment:
There's no guideline and even no common practice, but "ignores"/"flags" seems to be a little bit more widespread.
If you don't feel it fits the current spec structure that good, feel free to keep as it, it's good enough already.

Reply:
I guess you are referring to the use of the working `registers` with the context above it.

Most of what I did I saw in other cops (factory bot ones). However, I can see there are 2 or 3 ways of doing things. It is normal for standards and common practices to change over time. It would be nice if the documentation was updated to reflect that, the best practices etc. 

But anyway, it was a nice experience altogether. It will help me shape new cops for lots of things.

PS: I used the flags/ignores paradigm, I hope it is ok.